modular_skyrat/modules/modular_implants/code/nifsoft_shop.dm

```diff
@@ -0,0 +1,34 @@
+/obj/machinery/vending/nifsoft
+	name = "NIFSoft Vendor"
+	desc = "Money can be exchanged here for NIFSofts and other NIF goods" //Placeholder description
+	icon = 'modular_skyrat/modules/modular_implants/icons/obj/machines/vendors.dmi'
+	icon_state = "proj"
+	density = FALSE
+	default_price = 400
+	extra_price = 750
```

Comment:
Don't we typically have prices going using crate values or something like that? If that's not the case for the vendors, disregard, then.

Reply:
Nah, there's other vendors that I've noticed that have done this.